backend/priv/repo/migrations/20210824081357_add_position_to_posts_categories.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Til.Repo.Migrations.AddPositionToPostsCategories do
+  use Ecto.Migration
+
+  def change do
+    alter table(:posts_categories) do
+      add :position, :integer, default: 1, null: false
```

Comment:
it would be great to add unique constraint that prevents duplicates for the current scope so no category positions are duplicated for given post, please note that constraint should be marked as DEFFERED in PG, otherwise during the update you might have trouble updating multiple positions at the same time